packages/schemix/src/schemix/exceptions.py

```
@@ -0,0 +1,33 @@
+"""Custom exceptions for the Schemix ORM."""
+
+from __future__ import annotations
+
+
+class SchemixError(Exception):
+    """Base exception for all Schemix ORM errors."""
+
+    pass
+
+
+class ConnectionError(SchemixError):
```

Pylint (3.3.7)

[warning] Redefining built-in 'ConnectionError'

(W0622)